jest.config.cjs

```diff
@@ -7,6 +7,7 @@ module.exports = {
       statements: 100,
     },
   },
+  maxWorkers: 1,
```

Comment:
this is to enable serialization for `bigint` by `jest`

https://github.com/jestjs/jest/issues/11617#issuecomment-1028651059